packages/esm-patient-banner-app/src/banner-tags/print-identifier-sticker-content.scss

```diff
@@ -0,0 +1,19 @@
+@use '@carbon/layout';
+@use '@carbon/type';
+@use '@openmrs/esm-styleguide/src/vars' as *;
+
+.printContainer {
+  background-color: $ui-01;
+}
+
+.pageBreak {
+  page-break-after: always;
+}
+
+.labelsContainer {
+  display: grid;
+  column-gap: 1.3rem;
```

Comment:
Shouldn't this be `1.25rem`, given the styles in carbon are often used in multiples of .125rem
Thanks!